README.md

```diff
@@ -1,25 +1,24 @@
-# Phyloreferencing Curation Workflow
-The Phyloreferencing curation workflow serves three main purposes:
+# Clade Ontology
 
-1. It provides a set of exemplar curated phyloreferences in JSON and OWL.
-2. It provides a test space for trying different approaches to generating phyloreferences from JSON to OWL, although these will be moved into their own repositories if they prove to be successes.
-3. It provides a test suite of phyloreferences along with expected resolved nodes, allowing reasoning to be continually tested as ontologies and software tools are updated.
+The Clade Ontology is an ontology of exemplar phyloreferences curated from peer-reviewed publications. Phyloreferences in this ontology include their verbatim clade definition and the phylogeny upon which they were initially defined. The ontology therefore acts as both a catalogue of computable clade definitions as well as a test suite of phyloreferences that can be tested to determine if each phyloreference resolves as expected. This ontology is expressed in the [Web Ontology Language (OWL)](https://en.wikipedia.org/wiki/Web_Ontology_Language) and is available for reuse under the terms of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
```

Comment:
Where's the Apache license coming from?? OBO recommends CC-BY. I would suggest CC0, unless we have good arguments as to why we cannot waive copyright (for example, because someone other than us has it), or why our terms of reuse have to include a legal requirement for attribution (as opposed to a scientific norm mandating proper citation).

Reply:
The Apache license is intended to apply to the software in this repository -- the phyx2owl script and the testing scripts -- which is clearly not what the text in the README file says. I've updated the README file to clarify that, and added a copy of the CC0 legal code into the `phyx` directory to make it clear that a separate license applies there.

As for the question of other authors' copyright, I don't think you could argue that the phylogeny (which is a series of assertions) or the raw phyloreference (which is a list of specifiers) count as creative works, so CC0 sounds great to me. We are currently recording information that helps to explain why the clade was defined in a particular way (e.g. "Because sister group relationships are better resolved and supported than basally-branching ingroup relationships within this clade, a branch-modified node-based definition was used to maximize definitional and compositional stability (Cantino et al., 2007).", see reference below). This is arguably copyrightable, but I think it's small enough that we could argue fair use or remove it on a case-by-case basis if there are specific complaints.

https://github.com/phyloref/clade-ontology/blob/187c78cdf71c163075f3a216e6f4c5caa46c3e8d/testcases_too_slow/Wojciechowski%2C%202013/paper.json#L74)

Reply:
So fair use permitting us to reuse the definition texts here does not mean they are in the public domain, and fair use does mean you can claim intellectual property rights. So I'm skeptical that we have the rights to release the clade definition texts into the public domain.

Maybe this needs some consultation first with corresponding experts. Given the legal uncertainty, I'd argue we cannot at this point attach a CC0 waiver if the definition texts are included, because once we do we've waived our rights and we can't just take that back.